FitTracker/app/src/main/java/com/pps/globant/fittracker/mvp/model/LoginModel.java

```diff
@@ -1,23 +1,42 @@
 package com.pps.globant.fittracker.mvp.model;
 
-import com.facebook.CallbackManager;
+import android.app.Activity;
+
+import com.pps.globant.fittracker.model.User;
 import com.pps.globant.fittracker.utils.FacebookLoginProvider;
-import com.squareup.otto.Bus;
 
 public class LoginModel {
-    private final Bus bus;
-    private final CallbackManager callbackManager;
+    private FacebookLoginProvider facebookLoginProvider;
+    private User activeUser;
 
-    public LoginModel(CallbackManager callbackManager, Bus bus) {
-        this.bus = bus;
-        this.callbackManager = callbackManager;
+    public LoginModel(FacebookLoginProvider facebookLoginProvider) {
+        activeUser = null;
```

Comment:
This is not neccesary, for default will be assigned to null

Reply:
done